src/components/tabs/Tabs.js

```diff
@@ -188,56 +189,6 @@ export default class TabsComponent extends NestedComponent {
     this.addClass(this.tabLinks[index], 'active')
       .addClass(this.tabLinks[index].tabLink, 'active')
       .addClass(this.tabs[index], 'active');
-
-    this.triggerChange();
-  }
-
-  /**
-   * Return all the components within all the tabs.
-   */
-  getAllComponents() {
```

Comment:
Why did you remove all this code? This is needed to validate the items within each tab.

Reply:
This code is not required when we render all components from all tabs on the build stage. Validity checking from NestedComponent, in this case, works well.

Reply:
Well, we need this code back for us to accept this pull request. Other people use this renderer and we cannot just rip out code because it does not fit your use case.

Reply:
This code is the main problem which this pull request will fix and component behaviour won't change for any users (not only for us). Do you have any auto tests or testers who can verify that nothing changed from a user side? Or you need a deep analysis of the solved problem to accept this request? Removing the build method was a mistake, but this code (getAllComponents and checkValidity) just not working and leads to a lot of bugs. Any attempts to correct this logic will be failed in my opinion.

Reply:
Ok, thanks for the response. We will take a look as soon as we can if your changes work for validation across tabs.